### ISSUE TITLE
Uncaught TypeError: Cannot read property 'destroy' of undefined when date picker is inside a repeater 

### ISSUE THREAD

**Issue by chan4lk** [ghiscoding/Aurelia-Bootstrap-Plugins#53]
I have following html inside my custom element.

```html 
<div repeat.for="paramInfo of definitionParamInfo">
                                    <!-- Date -->
                                    <div class="${paramInfo.formGroupClass}" if.bind="paramInfo.DisplayType === 'date'">
                                        <label class="${paramInfo.IsOptional? '' : 'label-required'}">${paramInfo.DisplayName}</label>
                                        <small>
                                            <span class="validation-message text-danger">${paramInfo.validationMessage}</span></small>
                                        <abp-datetime-picker
                                        icon-base="font-awesome"
                                        class="form-control-danger form-control-success  form-control-sm  date-picker-wrapper"
                                        ></abp-datetime-picker>
                                        <small class="text-muted">${paramInfo.HelpText}</small>
                                    </div>

                                </div>
```
And I get following error when I load same custom element with different params.

``` javscript 
Uncaught` TypeError: Cannot read property 'destroy' of undefined
    at AbpDatetimePickerCustomElement.detached (index.js:21)
    at Controller.detached (vendor-bundle.js:31676)
    at View.detached (vendor-bundle.js:29733)
    at ViewSlot.detached (vendor-bundle.js:30081)
    at View.detached (vendor-bundle.js:29738)
    at removeAction (vendor-bundle.js:30026)
    at ViewSlot.removeAll (vendor-bundle.js:30049)
    at Repeat.removeAllViews (vendor-bundle.js:33952)
    at NullRepeatStrategy.instanceChanged (vendor-bundle.js:34062)
    at Repeat.itemsChanged (vendor-bundle.js:33833)
    at BehaviorPropertyObserver.selfSubscriber (vendor-bundle.js:31866)
    at BehaviorPropertyObserver.call (vendor-bundle.js:31730)
    at BehaviorPropertyObserver.setValue (vendor-bundle.js:31710)
    at Repeat.descriptor.set [as items] (vendor-bundle.js:31821)
    at Object.setValue (vendor-bundle.js:18744)
    at Binding.updateTarget (vendor-bundle.js:19998)```

**Comment by ghiscoding** [comment]
you might want to read previous issue #50

**Comment by ghiscoding** [comment]
Can you try latest version `1.3.0`, there was some improvements made for using it inside a repeater

**Comment by ghiscoding** [comment]
I forgot to mention that this was fixed in commit 71d6bad and released on [NPM](https://www.npmjs.com/package/aurelia-bootstrap-datetimepicker) as version `1.3.2`
